index.html

```diff
@@ -500,22 +500,6 @@ <h3>Classes</h3>
 Specifies that the credential is a Certificate of Citizenship credential.
             </td>
           </tr>
-          <tr>
-            <td>
-<a href="#OpticalBarcodeCredential">OpticalBarcodeCredential</a>
```

Comment:
I'm unsure how to handle these removed items.  On one hand, the v2 context is linking to them here.  On the other, the term definitions don't exist and these are broken references in the rendered spec.

How about since those terms were somewhat temporary, and there never were anchors for them here, that they be removed as done here, but also add a note in the README that v2 context is deprecated and includes some undefined terms.  (I'm not sure if they should even be listed.)

Reply:
I turned this into an issue so we can proceed and merge this PR and handle the issue afterwards.